addons/web_editor/static/src/js/editor/odoo-editor/src/commands/commands.js

```diff
@@ -676,7 +676,9 @@ export const editorCommands = {
         }
         referenceColumn.forEach((cell, rowIndex) => {
             const newCell = document.createElement('td');
-            newCell.append(document.createElement('br'));
+            const p = document.createElement('p');
+            p.append(document.createElement('br'));
```

Comment:
I think this fix and all the test changes that are related to it should be in their own commit as it is not really linked to the size reset.